app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    path = os.path.dirname(command[2])
```

Comment:
Add some checks to confirm that the command is correct and source file exists, if not - raise error